gems/aws-sdk-core/spec/spec_helper.rb

```diff
@@ -33,6 +33,13 @@ def initialize(_config = Config.new, _options = {})
   Config = Struct.new(:region, keyword_init: true)
 end
 
+module Aws
```

Comment:
Do you need defined? check here as well?

Reply:
There is no consequence for redefining these - however, it becomes a tricky order issue to check defined? We would need to add a `unless defined?(Aws::Crt::Checksums) && defined?(Aws::Crt::NextFeature) && defined?(Aws::Crt::Feature3)` for everything we add here...